site/no-nb/index.html

```diff
@@ -0,0 +1,669 @@
+<!DOCTYPE html>
+<!--
+  ~ Copyright (c) 2020 Goldman Sachs and others
```

Comment:
Please update the copyright according to your ECA.

Reply:
Done, thanks!